### PR TITLE
community: Bulk LocalAIEmbeddings

### PR DESCRIPTION
**Pull request by mkhludnev**
Thank you for contributing to LangChain!

- [x] **PR title**:

- [x] **PR message**: 
    - **Description:** single bulk request for embed_documents()
    - **Issue:** https://github.com/langchain-ai/langchain/discussions/22554
    - **Dependencies:** this pr on top of https://github.com/langchain-ai/langchain/pull/22399

- [x] **Add tests and docs**: integration test with vcr

- [x] **Lint and test**: 

Additional guidelines:

### CODE REVIEW

**Comment by isahers1** [comment]
@baskaryan This lgtm - I don't really understand the updates to the YAML file but other than that it makes sense

**Comment by mkhludnev** [comment]
YAMLs are requests and responses recorded by vcr for test stub.

File: libs/community/langchain_community/embeddings/localai.py [hunk 1]
```diff
@@ -135,15 +135,30 @@ class LocalAIEmbeddings(BaseModel, Embeddings):
...
     """
 
     client: Any  #: :meta private:
+    async_client: Any  #: :meta private:
     model: str = "text-embedding-ada-002"
     deployment: str = model
     openai_api_version: Optional[str] = None
     openai_api_base: Optional[str] = None
-    # to support explicit proxy for LocalAI
-    openai_proxy: Optional[str] = None
```
Comment (baskaryan) [inline libs/community/langchain_community/embeddings/localai.py:146]: why delete this?

**Comment by efriis** [comment]
has been in draft for a long time. will close, and feel free to reopen if relevant!

This might add a net-new community integration or feature, which has been replaced by dedicated integration packages. If reopened would recommend reopening with just docs updates, as well as registering your package in `libs/packages.yml`! We'll be able to review simple PRs that only modify these two things much faster :)

Here's the guide, and if you have questions, feel free to leave them in the comments on those pages so others can see them! https://python.langchain.com/docs/contributing/how_to/integrations/

**Comment by mkhludnev** [comment]
Available in new integration package since https://pypi.org/project/langchain-localai/0.1.2/

**Pull request closed without merging**